src/Tools/DynamoShapeManager/Utilities.cs

```diff
@@ -450,7 +447,11 @@ private static IEnumerable GetAsmInstallations(string rootFolder)
 
 
             var methodParams = new object[] { ProductsWithASM, ASMFileMask };
-            return installationsMethod.Invoke(null, methodParams) as IEnumerable;
+            var installs  = installationsMethod.Invoke(null, methodParams) as IEnumerable;
+
+            //filter install locations missing tbb and tbbmalloc.dll
+            return installs.Cast<KeyValuePair<string, Tuple<int, int, int, int>>>().
+                Where(x => Directory.EnumerateFiles(x.Key, "tbb*.dll").Count() >1);
```

Comment:
I think we can make this check more strict by specifically checking for both `tbb.dll` and `tbbmalloc.dll` just so we don't have any other surprises like `tbbX.dll` being present but neither being one or the other above.

Reply:
okay, will do.